device/mellanox/x86_64-mlnx_msn2700-r0/Mellanox-SN2700-D48C8/port_config.ini

```diff
@@ -1,57 +1,57 @@
-Ethernet14      14,15                  50000        etp4b
-Ethernet16      16,17                  50000        etp5a
-Ethernet18      18,19                  50000        etp5b
-Ethernet20      20,21                  50000        etp6a
-Ethernet22      22,23                  50000        etp6b
-Ethernet24      24,25,26,27            100000       etp7
-Ethernet28      28,29,30,31            100000       etp8
-Ethernet32      32,33,34,35            100000       etp9
-Ethernet36      36,37,38,39            100000       etp10
-Ethernet40      40,41                  50000        etp11a
-Ethernet42      42,43                  50000        etp11b
-Ethernet44      44,45                  50000        etp12a
-Ethernet46      46,47                  50000        etp12b
-Ethernet48      48,49                  50000        etp13a
-Ethernet50      50,51                  50000        etp13b
-Ethernet52      52,53                  50000        etp14a
-Ethernet54      54,55                  50000        etp14b
-Ethernet56      56,57                  50000        etp15a
-Ethernet58      58,59                  50000        etp15b
-Ethernet60      60,61                  50000        etp16a
-Ethernet62      62,63                  50000        etp16b
-Ethernet64      64,65                  50000        etp17a
-Ethernet66      66,67                  50000        etp17b
-Ethernet68      68,69                  50000        etp18a
-Ethernet70      70,71                  50000        etp18b
-Ethernet72      72,73                  50000        etp19a
-Ethernet74      74,75                  50000        etp19b
-Ethernet76      76,77                  50000        etp20a
-Ethernet78      78,79                  50000        etp20b
-Ethernet80      80,81                  50000        etp21a
-Ethernet82      82,83                  50000        etp21b
-Ethernet84      84,85                  50000        etp22a
-Ethernet86      86,87                  50000        etp22b
-Ethernet88      88,89,90,91           100000        etp23
-Ethernet92      92,93,94,95           100000        etp24
-Ethernet96      96,97,98,99           100000        etp25
-Ethernet100     100,101,102,103       100000        etp26
-Ethernet104     104,105                50000        etp27a
-Ethernet106     106,107                50000        etp27b
-Ethernet108     108,109                50000        etp28a
-Ethernet110     110,111                50000        etp28b
-Ethernet112     112,113                50000        etp29a
-Ethernet114     114,115                50000        etp29b
-Ethernet116     116,117                50000        etp30a
-Ethernet118     118,119                50000        etp30b
-Ethernet120     120,121                50000        etp31a
-Ethernet122     122,123                50000        etp31b
-Ethernet124     124,125                50000        etp32a
-Ethernet126     126,127                50000        etp32b
+# name          lanes                  speed        alias       index
```

Comment:
Sfputil is currently expecting the `index` column to be the fourth column, as it still expects columns to be in a particular order. Sfputil needs to be refactored to use portconfig.py instead of parsing port_config.ini directly. Until that change is made, the `index` column needs to be the fourth column.

`index` needs to

Reply:
I forgot I made that change :). So this should work. However, sfputil still needs to be refactored to use portconfig.py instead of parsing port_config.ini directly in the future.